README.md

```diff
@@ -133,7 +133,55 @@ We also provide nightly Conda packages built from the HEAD of our latest develop
 
 See the [RAPIDS release selector](https://rapids.ai/start.html#get-rapids) for more OS and version info.
 
-### Install from Source
+### Install with pip
+
+To install via pip:
+> **Note** cuSpatial is supported only on Linux or [through WSL](https://rapids.ai/wsl2.html), and with Python versions 3.9 and later
+
+The cuSpatial pip packages can be installed from NVIDIA's PyPI index:
+
+```shell
+# If using driver 525+, with support for CUDA Toolkit 12.0+
+pip install --extra-index-url=https://pypi.nvidia.com cuspatial-cu12
+
+# If using driver 450.80+, with support for CUDA Toolkit 11.2+
+pip install --extra-index-url=https://pypi.nvidia.com cuspatial-cu11
+
+# Or do this if you're unsure which CUDA Toolkit is supported by your driver:
```

Comment:
This is just cool! Going to use it in the docs.rapids.ai/install page I think!

Reply:
There might be a better way to get the CTK major version for the current driver, this is just something I whipped up on short notice.

Reply:
This looks eerily similar to something I just wrote yesterday: https://github.com/rapidsai/cudf/pull/13478/files#diff-76e052f6896a2d6de32795d5b94531dc822e5fc452495f2a1ba2e21ad2ee4970R14

Reply:
Yeah it is, but it uses `nvidia-smi` (which comes with the driver) instead of `nvcc`.